low-level/sensors/impl/centos_7/systemd_watchdog.py

```diff
@@ -1415,9 +1415,36 @@ def _get_drive_fault_info(self, path):
             cmd = f"sudo smartctl -d scsi -A {self._drive_by_device_name[path]}"
         else:
             cmd = f"sudo smartctl -A {self._drive_by_device_name[path]}"
-        response, _ = self._run_command(cmd)
+        response, _, _ = self._run_command(cmd)
         return response
 
+    def _is_local_drive(self, object_path):
+        """
+        Detect Node server local drives using Hdparm tool.
+        Hdparm tool give information only for node drive.
+        For external JBOD/virtual drives it will give output as:
+        "SG_IO: bad/missing sense data, sb[]:  70 00 05 00 00 00
+        00 0e 00 00 00 00 20 00 00 00 00 00 00 00 00 00 00 00 00 00 00
+        00 00 00 00 00\n HDIO_GET_IDENTITY failed: Invalid argument"
+        Hdparm does not have support for NVME drives, for this drives it gives o/p as:
+        "failed: Inappropriate ioctl for device"
+        """
+        ENCL_DISK_ERR = "SG_IO: bad/missing sense data, sb[]:  70 00 05" + \

```

Comment:
>SG_IO: bad/missing sense data

Limit check till above line and dont include the hex data, which may differ for different drives possibly and sspl check for local drive may fail.

Reply:
Done.